.storybook/webpack.config.ts

```diff
@@ -67,16 +67,16 @@ const webpackConfig = ({config}: {config: Configuration}) => {
 
     // Necessary to overwrite the values in the existing DefinePlugin hardcoded to the Config staging values
     const definePluginIndex = config.plugins.findIndex((plugin) => plugin instanceof DefinePlugin);
-    if (definePluginIndex !== -1 && config.plugins[definePluginIndex] instanceof DefinePlugin) {
-        const definePlugin = config.plugins[definePluginIndex] as DefinePlugin;
+    if (definePluginIndex !== -1 && config.plugins.at(definePluginIndex) instanceof DefinePlugin) {
+        const definePlugin = config.plugins.at(definePluginIndex) as DefinePlugin;
         if (definePlugin.definitions) {
             definePlugin.definitions.__REACT_WEB_CONFIG__ = JSON.stringify(env);
         }
     }
     config.resolve.extensions = custom.resolve.extensions;
 
     const babelRulesIndex = custom.module.rules.findIndex((rule) => rule.loader === 'babel-loader');
-    const babelRule = custom.module.rules[babelRulesIndex];
+    const babelRule = custom.module.rules.at(babelRulesIndex);
```

Comment:
This update changes the logic, cause `.at()` accepts negative integers while `[]` not.

![image](https://github.com/user-attachments/assets/e01e39dd-f6d6-4d2f-bbdb-b0cd2dc6105e)

So if `babelRulesIndex` is a `-1` -> `babelRule` can have a value instead of undefined.